core/src/main/java/nu/yona/server/analysis/service/ActivityService.java

```diff
@@ -167,7 +165,7 @@ private void createAndSaveInactivityDays(UserAnonymizedDTO userAnonymized, Set<G
 	private long getTotalPageableItems(UserAnonymizedDTO userAnonymized, ChronoUnit timeUnit)
 	{
 		long activityMemoryDays = yonaProperties.getAnalysisService().getActivityMemory().toDays();
-		Optional<ZonedDateTime> oldestGoalCreationTime = userAnonymized.getOldestGoalCreationTime();
+		Optional<LocalDateTime> oldestGoalCreationTime = userAnonymized.getOldestGoalCreationTime();
 		long activityRecordedDays = oldestGoalCreationTime.isPresent() ? (Duration
 				.between(oldestGoalCreationTime.get(), ZonedDateTime.now(ZoneId.of(userAnonymized.getTimeZoneId()))).toDays() + 1)
```

Comment:
Does the `Duration.between` convert the second argument `ZonedDateTime` to local date time in UTC correctly? I am not completely sure from the documentation...